frontend/public/webpack.config.js

```diff
@@ -21,8 +21,8 @@ module.exports = function (env, argv) {
     output: {
       path: path.resolve(__dirname, "build"),
       ...(isProduction ? {
-        filename: "[name]-[chunkhash].js",
-        chunkFilename: "[id]-[chunkhash].js",
+        filename: "[name]-[contenthash].js",
+        chunkFilename: "[id]-[contenthash].js",
```

Comment:
A "chunk" is all the files associated with a particular entrypoint (our entrypoints are `root`, `header`, `style`, `django`, `requirementsAdmin`). Typically this is css+js. 
-`chunkhash` is the hash of the chunk
- `contenthash` is a per-file hash

Previously: css change --> chunk hash changes --> js filename changes, which was unnecessary.

This doesn't really make much of a difference for mitxonline, since almost all CSS is in its own entrypoint. But this is a simple change, and is the recommended approach based on https://webpack.js.org/guides/caching/